tests/test_launcher_remote.py

```diff
@@ -0,0 +1,64 @@
+"""Test the PyPIM integration."""
+from unittest.mock import create_autospec
```

Comment:
Why ``unittest``? I cannot even see when this is installed in pymapdl

Reply:
`unittest` is a standard Python library: https://docs.python.org/3/library/unittest.html

That being said, we should be using `pytest` instead.

Reply:
Yes, fortunately pytest is very unopinionated on how the assertion are done, only on how tests are declared. Here I created the mocks using the built-in unittest (pytest does not have any specific tooling for that), but injected them with the `monkeypatch` fixture from pytest.

Reply:
There is ``pytest-mock`` but yeah... another package.